Small change to Ye(rho)

srichers opened the pull request:
Given the M1 capabilities of the code, I'm not sure you'll keep using Ye(rho), but this fixes a tiny issue I had when reading in Ye(rho) profiles. I also have a commit in there that I got from you as an email attachment (M1 implicit timestepping fix). Github does not see any difference between our codes in that area, so I presume you must have committed the same file yourself.

evanoconnor commented:
Hi Sherwood,

Can we split the ye(rho) out from the nulib changes?

Also, regarding the nulib changes, are you sure this is backwards compatible with using a table?  nulib.a is not something generated from the current nulibtable directory, rather nulibtable.a is

Evan

srichers commented:
Hi Evan,

Yeah, I also changed the makefile in the GR1D nulibtable directory to call
the library "nulib.a" instead of "nulibtable.a". It's a hack, since it
doesn't contain the same content as nulib.a produced by NuLib. Let me know
if there is a better way to do it.

If NULIBDIR is not defined, it defaults to building from
GR1D/src/nulibtable and works fine. However, this does not work with
HAVE_MC==1

Sherwood Richers

On Sat, Sep 9, 2017 at 1:37 AM, evanoconnor <notifications@github.com>
wrote:

> Hi Sherwood,
>
> Can we split the ye(rho) out from the nulib changes?
>
> Also, regarding the nulib changes, are you sure this is backwards
> compatible with using a table? nulib.a is not something generated from the
> current nulibtable directory, rather nulibtable.a is
>
> Evan
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/evanoconnor/GR1D/pull/12#issuecomment-328256207>, or mute
> the thread
> <https://github.com/notifications/unsubscribe-auth/AE54fE_5GnKL41hpYTc6KFUyJxlT_RCmks5sgiQTgaJpZM4PKSPC>
> .
>

srichers commented:
ugh, sent too soon. Anyway, it doesn't work with HAVE_MC==1 because Sedonu
requires more from NuLib than GR1D/src/nulibtable provides.

Sherwood Richers

On Sat, Sep 9, 2017 at 12:05 PM, Sherwood Richers <sarichers@gmail.com>
wrote:

> Hi Evan,
>
> Yeah, I also changed the makefile in the GR1D nulibtable directory to call
> the library "nulib.a" instead of "nulibtable.a". It's a hack, since it
> doesn't contain the same content as nulib.a produced by NuLib. Let me know
> if there is a better way to do it.
>
> If NULIBDIR is not defined, it defaults to building from
> GR1D/src/nulibtable and works fine. However, this does not work with
> HAVE_MC==1
>
> Sherwood Richers
>
> On Sat, Sep 9, 2017 at 1:37 AM, evanoconnor <notifications@github.com>
> wrote:
>
>> Hi Sherwood,
>>
>> Can we split the ye(rho) out from the nulib changes?
>>
>> Also, regarding the nulib changes, are you sure this is backwards
>> compatible with using a table? nulib.a is not something generated from the
>> current nulibtable directory, rather nulibtable.a is
>>
>> Evan
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/evanoconnor/GR1D/pull/12#issuecomment-328256207>, or mute
>> the thread
>> <https://github.com/notifications/unsubscribe-auth/AE54fE_5GnKL41hpYTc6KFUyJxlT_RCmks5sgiQTgaJpZM4PKSPC>
>> .
>>
>
>

srichers commented:
Also, I'm not sure what you mean by split out the changes, but you are
welcome to take all or none of the modifications. I'm just offering them in
case you find them useful. And now, in my scatterbrained state, I realize
that I did not sent the MC modifications yet, so my statement about HAVE_MC
is meaningless to you at this point.

Sherwood Richers

On Sat, Sep 9, 2017 at 12:06 PM, Sherwood Richers <sarichers@gmail.com>
wrote:

> ugh, sent too soon. Anyway, it doesn't work with HAVE_MC==1 because Sedonu
> requires more from NuLib than GR1D/src/nulibtable provides.
>
> Sherwood Richers
>
> On Sat, Sep 9, 2017 at 12:05 PM, Sherwood Richers <sarichers@gmail.com>
> wrote:
>
>> Hi Evan,
>>
>> Yeah, I also changed the makefile in the GR1D nulibtable directory to
>> call the library "nulib.a" instead of "nulibtable.a". It's a hack, since it
>> doesn't contain the same content as nulib.a produced by NuLib. Let me know
>> if there is a better way to do it.
>>
>> If NULIBDIR is not defined, it defaults to building from
>> GR1D/src/nulibtable and works fine. However, this does not work with
>> HAVE_MC==1
>>
>> Sherwood Richers
>>
>> On Sat, Sep 9, 2017 at 1:37 AM, evanoconnor <notifications@github.com>
>> wrote:
>>
>>> Hi Sherwood,
>>>
>>> Can we split the ye(rho) out from the nulib changes?
>>>
>>> Also, regarding the nulib changes, are you sure this is backwards
>>> compatible with using a table? nulib.a is not something generated from the
>>> current nulibtable directory, rather nulibtable.a is
>>>
>>> Evan
>>>
>>> —
>>> You are receiving this because you authored the thread.
>>> Reply to this email directly, view it on GitHub
>>> <https://github.com/evanoconnor/GR1D/pull/12#issuecomment-328256207>,
>>> or mute the thread
>>> <https://github.com/notifications/unsubscribe-auth/AE54fE_5GnKL41hpYTc6KFUyJxlT_RCmks5sgiQTgaJpZM4PKSPC>
>>> .
>>>
>>
>>
>
